src/utils/racePrint.js

```diff
@@ -0,0 +1,14 @@
+import { MissionUtils } from "@woowacourse/mission-utils";
+import { MESSAGES } from "../constants/messages.js";
+
+const racePrint = (RACE) => {
+  Object.entries(RACE).forEach(([CAR, PROGRESS]) => {
+    // pobi : -
+    const RACEING = MESSAGES.PROGRESS_BAR.repeat(PROGRESS);
+    MissionUtils.Console.print(`${CAR} : ${RACEING}`);
```

Comment:
```suggestion
    const RACING = MESSAGES.PROGRESS_BAR.repeat(PROGRESS);
    MissionUtils.Console.print(`${CAR} : ${RACING}`);
```
변수명에 스펠링 오류가 있는 것 같습니당!

Reply:
이걸 보시다니.. 정말 꼼꼼히 봐주셨네요  감사합니당